modules/tinymce/src/plugins/link/main/ts/ui/Controls.ts

```diff
@@ -95,7 +95,7 @@ const setupContextToolbars = (editor: Editor): void => {
       onSetup: Actions.toggleActiveState(editor)
     },
     label: 'Link',
-    predicate: (node) => !!Utils.getAnchorElement(editor, node) && Options.hasContextToolbar(editor),
+    predicate: (node) => Utils.getAnchorElement(editor, node).isSome() && Options.hasContextToolbar(editor),
```

Comment:
We seem to be using `Utils.getAnchorElement(editor, node).isSome()` quite a bit/have duplicated this in a number of places. Instead of doing all this duplication, can we move this to a helper function inside `Utils` as well? For example:
```ts
const isInAnchor = (editor: Editor, selectedElm?: Element): boolean =>
  getAnchorElement(editor, selectedElm).isSome()
```

Reply:
Done!